test/EventStore.Client.Tests.Common/.env

```diff
@@ -1,19 +1,11 @@
 ES_CERTS_CLUSTER=./certs-cluster
 ES_DOCKER_TAG=ci
 
-#EVENTSTORE_CLUSTER_SIZE=3
-#EVENTSTORE_INT_TCP_PORT=1112
+#EVENTSTORE_MEM_DB=true
 #EVENTSTORE_HTTP_PORT=2113
+#EVENTSTORE_LOG_LEVEL=Information
+#EVENTSTORE_DISABLE_LOG_FILE=true
+#EVENTSTORE_RUN_PROJECTIONS=None
+#EVENTSTORE_START_STANDARD_PROJECTIONS=true
 #EVENTSTORE_TRUSTED_ROOT_CERTIFICATES_PATH=/etc/eventstore/certs/ca
-#EVENTSTORE_DISCOVER_VIA_DNS=false
-#EVENTSTORE_ENABLE_EXTERNAL_TCP=false
-#EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP=true
-#EVENTSTORE_STREAM_EXISTENCE_FILTER_SIZE=10000
```

Comment:
this line is important to avoid creating a 256mb file (unless we are setting the stream existence filter size somewhere else now)

Reply:
we are adding this to the default settings of both node and cluster fixtures